force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +229,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result='';
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        for (Integer i=1; i<=repeatCount; i++) {
+            result += inputString; 
             // If it is not the last iteration, add a semicolon to separate the strings
+            if (i<repeatCount) {
+                result += '; ';
+            }
         }
```

Comment:
Another way to do this is to wait until your for loop is complete and then remove the last semicolon